### PR TITLE
Triggered

### PR DESCRIPTION
**Pull request by mathiasbno**

### CODE REVIEW

**Comment by jgorset** [comment]
Haha, thanks dude.

File: public/quotes.js [hunk 1]
```diff
@@ -179,7 +179,7 @@ var quotes = [
     quote: "I don't really like to party",
     author: "Johannes"
   },
-    {
+  {
     quote: "Fuckings box!",
     author: "Markus"
   },
```
Comment (RamonGilabert) [inline public/quotes.js:185]: Does the JSON work with this comma at the end?

Comment (toothfairy) [reply public/quotes.js:185]: Need new pull request with all review process for that

Comment (mathiasbno) [reply public/quotes.js:185]: No :(

**Pull request merged**